pkg/oc/cli/rollback/rollback.go

```diff
@@ -190,8 +188,8 @@ func (o *RollbackOptions) Validate() error {
 	if o.appsClient == nil {
 		return fmt.Errorf("oc must not be nil")
 	}
-	if o.kc == nil {
-		return fmt.Errorf("kc must not be nil")
+	if o.kubeClient == nil {
+		return fmt.Errorf("kubeInternalClient must not be nil")
 	}
 	if o.getBuilder == nil {
 		return fmt.Errorf("getBuilder must not be nil")
```

Comment:
I don't think this validation check is necessary - we don't do it in any other commands

Reply:
yeah, when I saw it I thought it is weird... will remove that